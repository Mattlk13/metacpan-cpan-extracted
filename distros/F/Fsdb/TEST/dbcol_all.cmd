prog='dbcol'
args='-a fullname'
cmp='diff -c -b '
in=TEST/dbrow_ex.in
