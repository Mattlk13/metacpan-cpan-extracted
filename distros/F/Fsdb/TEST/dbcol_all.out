#fsdb fullname account passwd uid gid homedir shell
John_Heidemann	johnh	*	2274	134	/home/johnh	/bin/bash
Greg_Johnson	greg	*	2275	134	/home/greg	/bin/bash
Root	root	*	0	0	/root	/bin/bash
# this is a simple database
#   | dbcol -a fullname
